ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -120,6 +120,19 @@ def display_if_datetime(value):
         return None
 
 
+def display_if_other_formats(value):
+    """dates/datetime values with other formats will be displayed in
+    ckanext.switzerland.date_picker_format
+    """
+    try:
+        for date_format in ALLOWED_DATE_FORMATS:
+            dt = datetime.strptime(value, date_format)
+            if isinstance(dt, datetime):
+                return isodate.strftime(dt, DATE_FORMAT)
```

Comment:
shouldn't it be 
`date_format` ? or where does `DATE_FORMAT` come from?

Reply:
DATE_FORMAT is the ogdch date format that we use - %d.%m.%Y. We want to output the date in this format rather than whatever format it was saved in.